### DISCUSSION TITLE
ARM version ? edit

### DISCUSSION THREAD

**Discussion by EricB1978** [home-assistant/operating-system#2296] [Q&A] [answered]
Hello, 
What version for an ARM version ? RPI4 ? 
I would like to install it in VM on my DELTA BOX FREE (French Internet Operator), and it is ARM64. 

Many Thanks.

**Comment by frenck** [comment, marked as answer]
arm64 == aarch64 :)

../Frenck

**Comment by EricB1978** [comment]
i just find a tutorial for an installation in ma freebox DELTA. All good.  

[https://www.domo-blog.fr/comment-installer-domotique-home-assistant-sur-freebox-delta/](url)

**Reply by frenck** [reply]
That is a third-party installation manual, which we do not recommend (as it is third-party). We recommend using the official documentation.
